js/modules/tamagotchi.js

```diff
@@ -5,6 +5,52 @@ export default class Tamagotchi {
 		this.energy = { value: 10, importance: 2 };
 		this.fun = { value: 10, importance: 4 };
 		console.log('Tamagotchi initialized');
+
+		const energyValue = '.energy';
```

Comment:
**nitpick:** You should not hardcode these values. You already pass them in `app.js` - what about mounting them to this constructor?:)